app/forms/teacher_interface/name_and_date_of_birth_form.rb

```diff
@@ -23,7 +23,8 @@ def save
   private
 
   def validate_date_of_birth
-    if date_of_birth.present? && date_of_birth >= 18.years.ago
+    if date_of_birth.present? &&
+         (date_of_birth >= 18.years.ago || date_of_birth <= 100.years.ago)
```

Comment:
Can we do `date_of_birth.between?`here? Also wondering if this could just be a standard rails inclusion validation?

Reply:
Good point, I've refactored to use `inclusion`.

Reply:
Actually, will that set the dates from when the app last started? I don't think it really matters that much here.

Reply:
Hmm true, I think it's okay, but something to keep an eye on...